src/components/SubmissionConfirmation.js

```diff
@@ -67,7 +67,7 @@ StartPayment.propTypes = {
  * @param {Function} onFailure Callback to invoke if the background processing result is failure.
  * @param {Function} onConfirmed Callback to invoke if the background processing result is success.
  */
-const SubmissionConfirmation = ({statusUrl, onFailure, onConfirmed}) => {
+const SubmissionConfirmation = ({statusUrl, onFailure, onConfirmed, submission, form}) => {
```

Comment:
`submission` prop is unused, and `form` prop should be added to the component prop types.